lib/std/debug.zig

```diff
@@ -262,6 +262,12 @@ pub fn panicExtra(trace: ?*const builtin.StackTrace, first_trace_addr: ?usize, c
                 defer held.release();
 
                 const stderr = io.getStdErr().writer();
+                if (builtin.single_threaded) {
+                    stderr.print("main thread panicked: ", .{}) catch os.abort();
```

Comment:
```suggestion
                    stderr.print("panic: ", .{}) catch os.abort();
```

"main thread panicked" implies both (1) there are or could be other threads and (2) in a multi-threaded build the panic message would distinguish between the main thread and other threads.